src/applications/check-in/day-of/pages/Landing.jsx

```diff
@@ -8,7 +8,7 @@ import recordEvent from 'platform/monitoring/record-event';
 import { getTokenFromLocation, URLS, goToNextPage } from '../utils/navigation';
 import { api } from '../api';
 import { tokenWasValidated, triggerRefresh } from '../actions';
-import { setCurrentToken, clearCurrentSession } from '../../utils/session';
+import { useSessionStorage } from '../../hooks/useSessionStorage';
```

Comment:
Just because this file was touched, We should update the loading component to be the web-component instead